### DISCUSSION TITLE
Unhandled Runtime Error

### DISCUSSION THREAD

**Discussion by Sphinxkkk** [WalletConnect/walletconnect-monorepo#4297] [Developer Support]
Unhandled Runtime Error
HttpRequestError: HTTP request failed.

Status: 401
URL: https://rpc.walletconnect.org/v1/?chainId=eip155:137&projectId=cd web3modalyarn dev
Request body: {"method":"eth_getBalance","params":["0x7713F9FDEB29AAaeA2AACd517212a9a8B4936462","latest"]}

Version: viem@1.21.3

Call Stack
<unknown>
httprequesterror: HTTP request failed.
Object.http
node_modules\viem\_esm\utils\rpc.js (41:0)
async fn
node_modules\viem\_esm\clients\transports\http.js (40:0)
async request
node_modules\viem\_esm\clients\transports\http.js (41:40)
async delay.count.count
node_modules\viem\_esm\utils\buildRequest.js (26:0)
async attemptRetry
node_modules\viem\_esm\utils\promise\withRetry.js (12:0)

What's going on? Please help me answer it. Thank you!

**Comment by glitch-txs** [comment]
Hi, could you share a minimal reproducible example and steps to reproduce the issue? Thanks!
